app/src/main/java/com/example/concordia_campus_guide/activities/PathsActivity.java

```diff
@@ -76,8 +80,8 @@ private void initComponents() {
         getBundleInformation();
         setBackButtonOnClickListener();
 
-        locationFragment.setDifferentInitialView(new LatLng(initialLocation.getCenterCoordinates().getLatitude(),
-                initialLocation.getCenterCoordinates().getLongitude()));
+        locationFragment.setDifferentInitialView(new LatLng(initialLocation.getLatitude(),
```

Comment:
this causes me issues when running the app

Reply:
it is linked to the error I mentioned in your PR.

Reply:
ok fixed it thanks